gpkit/substitution.py

```diff
@@ -70,7 +70,40 @@ def vectorsub(subs, var, sub, varset):
         subs[var] = sub
 
 
-def substitution(varlocs, varkeys, exps, cs, substitutions, val=None):
+def separatesubs(nomial, substitutions):
```

Comment:
That's because all the other goodness is in get_constants. They should be
combined better, though.